relayer/application_relayer.go

```diff
@@ -246,8 +206,13 @@ func (r *ApplicationRelayer) relayMessage(
 
 	// sourceWarpSignatureClient is nil iff the source blockchain is configured to fetch signatures via AppRequest
 	if r.sourceWarpSignatureClient == nil {
+		// TODO: do we actually want to pass the pointer here or adapt the interface?
```

Comment:
is this TODO decided on, and if not, should we handle it before the merge

Reply:
Sorry I removed this on a different branch. I will add it here as well. There is no longer interface since the both applications use the same aggregator and AppRequestNetwork.